torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11316,6 +11316,52 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
            check_batched_forward_grad=False,
            gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
            sample_inputs_func=sample_inputs_adaptive_max_pool3d),
+    OpInfo(
+        "nn.functional.alpha_dropout",
+        op=lambda input, *args, **kwargs:
+            wrapper_set_seed(torch.nn.functional.alpha_dropout, input, *args, **kwargs),
```

Comment:
good news, for `without_train` you don't need this seed wrapper, and skips that are caused by it.